src/typings.ts

```diff
@@ -40,6 +41,7 @@ export interface IClusterMarkerProps {
   children: ReactNode;
   clusterId: number;
   onClusterMarkerPress: (clusterClickEvent: IClusterClickEvent) => void;
+  customMarkerProps?: object;
```

Comment:
Could you add information about this prop to Readme.md?
clusterMarkerProps - object - undefined - Applies additional props to ClusterMarker